utils/constants.ts

```diff
@@ -18,6 +18,9 @@ export const LINEAR = {
         todo: "Todo",
         done: "Done",
         canceled: "Canceled"
+    },
+    PUBLIC_QUERY_HEADERS: {
+        "public-file-urls-expire-in": "86400"
```

Comment:
I'm not sure what the behavior of the GitHub cache is, _if_ we see that the images start to break after a day then you may want to increase this value.